go/client/logclient.go

```diff
@@ -308,3 +309,11 @@ func (c *LogClient) GetSTH() (sth *ct.SignedTreeHead, err error) {
 	sth.TreeHeadSignature = *ds
 	return
 }
+
+// GetSTHConsistency retrieves the consistency proof between two snapshots.
+func (c *LogClient) GetSTHConsistency(first, second uint64) ([][]byte, error) {
```

Comment:
I did. That would mean returning the `getConsistencyProofResponse` object, which would also mean exporting it. This seemed cleaner.